Comp2023/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -538,6 +542,70 @@ public void drive(Translation2d translation, double rotation, boolean fieldRelat
     }
   }
 
+  public void driveBalance( )
+  {
+    pitch = m_pigeon.getUnadjustedPitch( ).getWPIRotation2d( ).getDegrees( );
+    DataLogManager.log("Balance Off By: " + pitch);
```

Comment:
This is going to spam the screen during balancing. We don't want this log message active all the time.

Reply:
It's OK for debugging--just not 100% of the time. So maybe comment it out or possibly slow the rate.